docs/content/en/docs/reference/clusterspec/oidc.md

```diff
@@ -39,36 +39,36 @@ spec:
     usernamePrefix: ""
 ```
 ### identityProviderRefs (Under Cluster)
-List of identity providers you want configured for the Cluster.
+List of identity providers you want configured for the cluster.
 Right now, only 1 provider of type `OIDCConfig` is supported.
 This would include a reference to the `OIDCConfig` object with the configuration below.
 
 ### clientId (required)
-* Description: ClientId defines the client ID for the OpenID Connect client
+* Description: Client ID for the OpenID Connect client.
 * Type: string
 ### groupsClaim (optional)
-* Description: GroupsClaim defines the name of a custom OpenID Connect claim for specifying user groups
+* Description: Name of a custom OpenID Connect claim for specifying user groups.
 * Type: string
 ### groupsPrefix (optional)
-* Description: GroupsPrefix defines a string to be prefixed to all groups to prevent conflicts with other authentication strategies
+* Description: String to be prefixed to all groups to prevent conflicts with other authentication strategies.
 * Type: string
 ### issuerUrl (required)
-* Description: IssuerUrl defines the URL of the OpenID issuer, only HTTPS scheme will be accepted
+* Description: URL of the OpenID issuer. Only HTTPS scheme will be accepted.
 * Type: string
 ### requiredClaims (optional)
-List of RequiredClaim objects listed below. 
+List of requiredClaim objects listed below. 
```

Comment:
The field name is `requiredClaims`, but the objects are of kind `RequiredClaim` so I think this is right.

Reply:
Fixed in next revision